react/features/base/config/functions.any.ts

```diff
@@ -85,6 +85,16 @@ export function getSourceNameSignalingFeatureFlag(state: IState) {
     return getFeatureFlag(state, FEATURE_FLAGS.SOURCE_NAME_SIGNALING) ?? true;
 }
 
+/**
+ * Selector used to get the SSRC-rewriting feature flag.
+ *
+ * @param {Object} state - The global state.
+ * @returns {boolean}
+ */
+export function getSsrcRewritingFeatureFlag(state: IState) {
+    return getFeatureFlag(state, FEATURE_FLAGS.SSRC_REWRITING);
```

Comment:
Do we need test if source-name signaling is explicitly disabled ? ssrc-rewriting is works only with source-name signaling right ?

Reply:
ok, fixed.